client/src/Firebase/firebase.js

```diff
@@ -0,0 +1,36 @@
+import app from 'firebase/app';
+import 'firebase/auth';
```

Comment:
is this file going to conflict with the other firebase.js in the other firebase related branch route?

Reply:
it shouldn't. but i definitely should have made sure I only did a git add of the files that changed for this branch.